external/float32/float32_stubs.c

```diff
@@ -0,0 +1,332 @@
+/**************************************************************************/
+/*                                                                        */
+/*                                 OCaml                                  */
+/*                                                                        */
+/*                       Pierre Chambart, OCamlPro                        */
+/*           Mark Shinwell and Leo White, Jane Street Europe              */
+/*                        Max Slater, Jane Street                         */
```

Comment:
This should probably retain the copyright
from upstream.

Reply:
Done